src/Octoshift/CodeScanningAlerts.cs

```diff
@@ -0,0 +1,20 @@
+namespace OctoshiftCLI
+{
+    public static class CodeScanningAlerts
```

Comment:
It's more customary for enums (and enum likes) to be singular as opposed to plural:
```suggestion
    public static class CodeScanningAlert
```

Reply:
I ended up with naming collisions with this and the DTO objects, this also applied to the SecretScanningAlert class that was the same, hence why I was using the pural here. I ended up renaming the DTO objects to be prefixed with Github as in the future there could be other objects from other Git hosting systems that need models.